invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js

```diff
@@ -28,6 +28,10 @@ export const Results = ({ currentResultsState = {} }) => {
   const { sortOptions, layoutOptions, paginationOptions, buildUID } =
     useContext(SearchConfigurationContext);
   const multipleLayouts = layoutOptions.listView && layoutOptions.gridView;
+
+  const handleMathjaxCallback = useCallback(() => {
```

Comment:
Are you sure that the `useCallback()` is needed? What about if you change page i.e. you rerender with different results without refreshing the page? Does `Mathjax.typeset()` is being called again (assuming that it needs to)?

Reply:
`Mathjax.typeset()` is getting called again and the formulas are printed correctly in case the component gets rerendered without reloading the page. I'm also assuming this is the correct behavior (correct me if im wrong)

Reply:
Minor: I think that the `useCallback` is an over-optimization here.